### PR TITLE
Create treatment and appointment models

### PR DESCRIPTION
**Pull request by froidautaj**
Models for both the treatment and appointment models with both their respected validations

### CODE REVIEW

File: app/models/appointment.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class Appointment < ApplicationRecord
+  belongs_to :user
+  belongs_to :treatment
+
+  validates_presence_of :treatment
```
Comment (Andy-Hoch) [inline app/models/appointment.rb:5]: Validations are not necessary

File: app/models/treatment.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class Treatment < ApplicationRecord
+  belongs_to :user
+  has_many :appointments
+  has_many :reviews
+
+  validates_presence_of :user, uniqueness: true
```
Comment (Andy-Hoch) [inline app/models/treatment.rb:6]: Validations are not necessary

File: db/migrate/20230821155313_create_treatments.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
+class CreateTreatments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :treatments do |t|
+      t.string :name
+      t.string :category
+      t.text :description
+      t.text :address
+      t.string :image_url
```
Comment (NicoleDeu) [inline db/migrate/20230821155313_create_treatments.rb:8]: We have to remove the column image_url, this will happen later with cloudinary

**Review by Andy-Hoch — CHANGES_REQUESTED** [review]
Update please

**Review by NicoleDeu — CHANGES_REQUESTED** [review]
Hey Froi, could you have a look at my comments?

**Pull request closed without merging**